test/VectorInstructions/insert_element_symbolic.c

```diff
@@ -0,0 +1,22 @@
+// REQUIRES: geq-llvm-3.4
```

Comment:
I'd rather keep these tests. They force someone who actually changes KLEE's behaviour to actually do something about the tests because they will break if support for symbolic insert/extract is added.

Also, testing KLEE is not the bottleneck in CI, building is. Running the tests is pretty fast and KLEE's test coverage if pretty poor, we need more tests, not less.